src/screens.ts

```diff
@@ -1,10 +1,11 @@
 // Dependencies
-const chalk = require('chalk');
-const Table = require('cli-table');
-const utils = require('./utils');
+import chalk from 'chalk';
+import Table from 'cli-table';
+import { Settings } from './types';
+import utils from './utils';
 
-// Helpful utility functions
-module.exports = function Constructor(currentSettings) {
+// Default export
+function screens(currentSettings: Settings) {
```

Comment:
This would be more consistent with other files...

```suggestion
export default function screens(currentSettings: Settings) {
```

You'll need to remove the `export default screens;` line down below, too.